### DISCUSSION TITLE
RE 2 VR works great but RE 3 first person will not work??

### DISCUSSION THREAD

**Discussion by davidflana** [praydog/REFramework#929] [General Questions]
Hello,
I put many hours into RE2 VR mod, and loved it!  I finished the game so I bought RE 3 and downloaded and installed the VR mod for that hoping for the same great experience.  Well, when the game play finally starts, I am in 3rd person, and can look around like in first person.  When I switch over to first person, I can no longer look around with my head.  It locks on and only moving the joysticks allows me to see other areas.  Also, my hands/arms do not work.....  I am using Quest 2 on a Legion 7i laptop with steamVR.  I have tried reloading the mod, I tried the non ray tracing version, nothing seems to work.  I go back and play RE2 just make sure.... it works great still!  Any ideas?  I REALLY want to continue my journey!  thanks!

**Comment by praydog** [comment]
Get past the first five minutes of the game

**Reply by davidflana** [reply]
that was the problem!  Thank you very much for your prompt reply.  The game is awesome, thank you for all the work you put in on it!
